src/styles/variables.scss

```diff
@@ -0,0 +1,19 @@
+$text-color: #060b35;
+$subtext-color: #616070;
+$button-color: #00acdc;
+$left-margin: 16px;
+$text-size: 12px;
+$font-family: 'Roboto', sans-serif;
+$border-color: white;
+$border-radius: 5px;
+$star-size: 16px;
+$buy-button-font-size: 14px;
+$buy-button-font-weight: 700;
+$stars-count: 5;
+$font-family: 'Roboto', sans-serif;
+$border-color: white;
+$border-radius: 5px;
+$star-size: 16px;
+$buy-button-font-size: 14px;
+$buy-button-font-weight: 700;
+$stars-count: 5;
```

Comment:
There are duplicate variable declarations in this file. The variables `$font-family`, `$border-color`, `$border-radius`, `$star-size`, `$buy-button-font-size`, `$buy-button-font-weight`, and `$stars-count` are defined twice with the same values. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid code duplication. Please remove the duplicate variable declarations.